glean_parser/markdown.py

```diff
@@ -121,6 +122,28 @@ def ping_data_reviews(
         return None
 
 
+def ping_review_title(data_url: str, index: int) -> str:
+    """
+    Return a title for a data review in a readable form.
```

Comment:
Can you also document the parameters passed into this function?

Reply:
Done! Thanks